android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
Just asking since i don't know android 😓shouldn't this kind of config match master? (meaning that we should update that branch as well)

Reply:
Yes, I believe so. I'll create a PR against master with that change later.